app/views/events/_event.html.erb

```diff
@@ -0,0 +1,14 @@
+<li>
+    <div>
+        <h2>Location: <%=link_to "#{event.location}", event_path(event) %></h2>
+
+        <div>
+            <p>Date: <%=event.date%></p>
+            <p>Creator: <%=event.creator.name%></p>
+        </div>
+    </div>
+    <%if logged_in?%>
+    <%= link_to "Attend event", attend_event_path(event), method: :post,
+                                       data: { confirm: "You sure?" } %>
+    <%end%>
```

Comment:
The Attend event link is still there even after attending an event.
To prevent this, we can create a helper method in `app/helpers/event_helpers.rb` like this:
```ruby
module EventsHelper
    def attending?(event)
        event.attendees.include? current_user
    end
end
```
Then invoke it in the views like this.
```ruby
<%if logged_in? && !attending?(event)%>
```